changelog.d/0-release-notes/pr-2059

```diff
@@ -0,0 +1 @@
+The optional file sharing team feature config now has a new required attribute: `lockStatus`. For more information refer to the `config-options` documentation. Prior to this change the file sharing team feature was not configurable for on-premise installations. It has now been added as an optional mapping to the `configmap.yaml` galley template.
```

Comment:
I think this entire file could be removed:
- for wire cloud, the config was not overridden anywhere, so nothing needs to be done during a release.
- for test backends, the config was not overridable to begin with, so no action is required.
- customers were not able to override this setting in the past, so no additional warning is needed here. Perhaps just include parts of this under 2-features and remove this extra release note file; and everyone interested can see "new feature, configurable fileSharing options", but nobody needs to worry about having to care about this if they are not interested in fileSharing options.